README.md

```diff
@@ -4,12 +4,12 @@
 [![Known Vulnerabilities](https://snyk.io/test/github/laniksj/slack-dark-mode/badge.svg?targetFile=/docs/Gemfile.lock)](https://snyk.io/test/github/laniksj/slack-dark-mode?targetFile=/docs/Gemfile.lock)
 
 ## Purpose
-Store scripts and Style sheets for Slack Dark Mode for macOS Mojave.  
-Dark Mode in Slack isn't available as of this writing.  
+Store scripts and Style sheets for Slack Dark Mode for macOS Mojave.
+Dark Mode in Slack isn't available as of this writing.
```

Comment:
This change breaks the formatting for README.md.  Please revert this one.

Reply:
What do you mean it breaks the formatting?  I'm not seeing the difference.  All it did was remove the white space (which is a setting i have in my vscode).

I can revert it sure, but not sure what the issue is exactly.

Reply:
OH i see, nevermind, got it.